ocs_ci/utility/utils.py

```diff
@@ -2915,29 +2915,100 @@ def get_zone_number(self):
         return prev
 
 
-def convert_device_size(unformatted_size, units_to_covert_to):
+def convert_device_size(unformatted_size, units_to_covert_to, convert_size=1000):
     """
     Convert a string representing a size to an int according to the given units
     to convert to
 
     Args:
         unformatted_size (str): The size to convert (i.e, '1Gi'/'100Mi')
         units_to_covert_to (str): The units to convert the size to (i.e, TB/GB/MB)
+        convert_size (int): set convert by 1000 or 1024
 
     Returns:
         int: The converted size
 
     """
     units = unformatted_size[-2:]
```

Comment:
If the unit to convert is `B` it will fail because it will take 2 characters as the suffix and one of them will be a digit.
byte_pattern = re.compile("[^A-Z]*B$")
if byte_pattern.match(unformatted_size):
     units = unformatted_size[-1:]
     abso = int(unformatted_size[:-1])
else:
     units = unformatted_size[-2:]
     abso = int(unformatted_size[:-2])

Reply:
The indentation is not working but you get my point...

Reply:
Since there is no `B` but you've changed it to 2 letters dismissing.